src/config.ts

```diff
@@ -2,10 +2,11 @@
  * Main application config.
  */
 
-const cliArgs = process.argv.slice(2);
+const cliArgs = process.argv.slice(4);
+const apiKey = cliArgs[0] === '--KEY' || cliArgs[0] === '--key' ? cliArgs[1] : null;
 
 
 export default Object.freeze({
     APP_PORT: process.env.APP_PORT || 8080,
-    ENV: cliArgs[0] === '--ENV' || cliArgs[0] === '--env' ? cliArgs[1] : null
+    OMDB_API_BASE_URL: `http://www.omdbapi.com/?apikey=${apiKey}`
```

Comment:
query params should not be added to the base url since the service might have other routes like `BASE_URL/foo`

Reply:
Thank you, indeed. Fixed.